boards/pic32-wifire/pic32_config_settings.c

```diff
@@ -0,0 +1,403 @@
+/*
```

Comment:
This has nothing to do with MIPS architecture. This is a weird quirk of the PIC32 implementation from Microchip (and there odd programming tools). These value map onto registers which overlay the flash, when a reflash occurs these register get 'erased' and must be programmed when a new firmware image is loaded and thus these values must exist at the correct addresses in the resultant elf file / binary image.
I agree its horrible but a necessity for PIC32 devices, Microchip in there own tools (XC32) have implemented a #pragma for these values which make it a bit nicer:
https://people.ece.cornell.edu/land/courses/ece4760/PIC32/PLIB_examples/plib_examples/uart/uart_basic/source/main.c
We don't have that ability in generic tools.